gradle.properties

```diff
@@ -1,3 +1,3 @@
 group=com.enonic.guide
 projectName=guide-testing-apps
-xpVersion=6.11.0
+xpVersion=7.0.0
```

Comment:
we update to latest XP API level (currently 7.7.0)

Reply:
no, we shouldn't. this is a guide, not a starter, so it should be 7.0.0